pym/bob/input.py

```diff
@@ -2212,6 +2225,9 @@ def coDet(r):
             self.__relocatable = self.__recipeSet.getPolicy('allRelocatable') \
                 or not self.__provideTools
 
+        if self.__jobServer is None:
```

Comment:
I'm sure there was class inheritance code before. Looks like it got lost in the last refactoring. Currently the property will not work when set in a (anonymous) class.